pkg/flexvol/flexvol_windows.go

```diff
@@ -0,0 +1,103 @@
+/*
+(c) Copyright 2017 Hewlett Packard Enterprise Development LP
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package flexvol
+
+import (
+	log "github.com/hpe-storage/common-host-libs/logger"
+	"github.com/hpe-storage/common-host-libs/util"
+	"os"
+	"path"
+	"strings"
+)
+
+func doMount(flexvolPath, dockerPath, dockerName, mountID string) error {
+	// Get volume object for a disk mounted by plugin.
```

Comment:
I think since we want to eventually use CHAPI for this, we should put the code in common-host-libs for now, right?

Just like the linux version was calling linux.functions(), we should move this to windows.functions().

Reply:
Yes it using powershell commands directly and later we would want to use CHAPI however the doMount() is only required for flexvolume driver. The code is right now bifurcated _linux and _windows. If we still want to move to common-host-libs , then we would want to move both flexvol_linux.go and flexvol_windows.go . Pls let me know .

Reply:
No, you don't have to move flexvol_linux and flexvol_windows?  Why not just move the powershell commands you are doing to a windows package in common-host-libs?  Then you simply call windows.function() to do what you need.

Reply:
I see, ok .

Reply:
@rgcostea , this is done. please review this and common-host-libs changes.